site/templates/application/application.jet

```diff
@@ -80,15 +80,16 @@
         window.Shift72CheckoutOptions = {
           baseUrl: window.location.origin,
           {{if useLocalDevCheckout}}checkoutDevServerUrl: '{{checkoutDevServer}}',{{end}}
-          options: {
           links: {
-              library: `${window.location.origin}{{routeToPath("/library.html")}}`,
-              termsAndConditions: `${window.location.origin}{{routeToPath("/page/terms-and-conditions/")}}`,
-              deviceCompatibility: `${window.location.origin}{{routeToPath("/page/help/")}}`,
-              help: `${window.location.origin}{{routeToPath("/page/help/")}}`,
-              signIn: `${window.location.origin}{{routeToPath("/signin.html")}}`,
-              signUp: `${window.location.origin}{{routeToPath("/signup.html")}}`,
-            }
+            library: `${window.location.origin}{{routeToPath("/library.html")}}`,
+            termsAndConditions: `${window.location.origin}{{routeToPath("/page/terms-and-conditions/")}}`,
+            deviceCompatibility: `${window.location.origin}{{routeToPath("/page/help/")}}`,
+            help: `${window.location.origin}{{routeToPath("/page/help/")}}`,
+            signIn: `${window.location.origin}{{routeToPath("/signin.html")}}`,
+            signUp: `${window.location.origin}{{routeToPath("/signup.html")}}`,
+          },
+          getLang: () => {
```

Comment:
It is for our site, but I wasn't sure if other sites that embed it could have a language switcher that supported changing language without page refresh. In saying that, they could just edit the options object: `window.Shift72CheckoutOptions.lang`. Happy to change if you feel it makes more sense.

Reply:
I think it's tidier. As I commented on the Checkout PR, if this pattern is generally needed, we could support `T | () => T` options for the embed library (and have them get automatically evaluated via a helper)